### ISSUE TITLE
Inverse_transform not working in RowTransform(StandardScaler())

### ISSUE THREAD

**Issue by jmrichardson** [alan-turing-institute/sktime#366]
**Describe the bug**
```python
scaler = RowTransformer(StandardScaler())
Xtrain = scaler.fit_transform(self.X_train)
Xtrain_inv = scaler.inverse_transform(Xtrain)
```
Inverse_transform does not work when applying RowTransformer(StandardScaler()).  Apparently, sktime does not store the fitted standard scaler for each series.  See [issue](https://github.com/sktime/sktime-dl/issues/62)

**Expected behavior**
<details>
I would have thought that the above code self.X_train and Xtrain_inv would be identical but they differ significantly.  
</details>

**Versions**
<details>

Windows-10-10.0.18362-SP0
Python 3.7.8 | packaged by conda-forge | (default, Jul 31 2020, 01:53:57) [MSC v.1916 64 bit (AMD64)]
NumPy 1.18.5
SciPy 1.4.1
sktime 0.4.1

</details>

**Comment by mloning** [comment]
This will be addressed in #420 where we rework the RowTransformer.

**Comment by mloning** [comment]
Closed by #420
